src/server/detail/save_stages_controller.cc

```diff
@@ -153,7 +153,7 @@ void RdbSnapshot::StartInShard(EngineShard* shard) {
 
 SaveStagesController::SaveStagesController(SaveStagesInputs&& inputs)
     : SaveStagesInputs{std::move(inputs)} {
-  start_time_ = absl::Now();
+  start_time_ = time(NULL);
```

Comment:
out of curiosity: why `time(NULL)` here - and then `time(nullptr)` in the next line of the diff?

what is the C++ Standard version dragonfly targets? Are their fixed coding rules?

Reply:
We use C++17, and yeah we should use `nullptr` rather than `NULL`, but old habits die hard :)